### PR TITLE
remove rubocop references in lieu of angellist-rubocop

### PR DESCRIPTION
**Pull request by stathis-alexander**
I'm not sure if anywhere is still using this or not.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### [0.1.8](https://github.com/angellist/eslint-config-angellist/compare/0.1.7...0.1.8) - 2024-12-04
```
Comment (FoodProduct) [inline CHANGELOG.md:3]: How about
```suggestion
### [0.2.0](https://github.com/angellist/eslint-config-angellist/compare/0.1.7...0.2.0) - 2024-12-04
```
?

**Review by FoodProduct — COMMENTED** [review]
I think we should do a minor version bump (I know there are no real rules around pre-1.0 versioning) otherwise makes sense to me.

File: CHANGELOG.md (hunk 1 above)
Comment (stathis-alexander) [reply CHANGELOG.md]: Good call, this is a breaking change.

**Review by FoodProduct — APPROVED** [review]
🚮

**Pull request merged**